src/main/java/application/WebAppContext.java

```diff
@@ -172,6 +175,16 @@ public static PropertySourcesPlaceholderConfigurer propertiesResolver() {
     }
 
     @Bean
+    public StatsDClient datadogStatsDClient() {
+        return new NonBlockingStatsDClient(
+                "formplayer.metrics",
+                "localhost",
+                8125,
+                new String[] {"tag:formplayer"}
```

Comment:
`tag` here is meant to be the name of the tag. Since the metric is already prefixed with `formplayer` this tag doesn't seem useful.